jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/tasklogs/TaskLogItemView.java

```diff
@@ -0,0 +1,153 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.ht.client.editors.tasklogs;
+
+import com.google.gwt.user.client.TakesValue;
+import org.jboss.errai.common.client.api.IsElement;
+import org.jboss.errai.common.client.dom.Div;
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.common.client.dom.Span;
+import org.jboss.errai.databinding.client.api.DataBinder;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.AutoBound;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.common.client.util.AbstractView;
+import org.jbpm.workbench.common.client.util.DateUtils;
+import org.jbpm.workbench.ht.client.resources.i18n.Constants;
+import org.jbpm.workbench.ht.model.TaskEventSummary;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+@Dependent
+@Templated
+public class TaskLogItemView extends AbstractView<TaskLogsPresenter>
+        implements TakesValue<TaskEventSummary>,
+                   IsElement {
+
+    Constants constants = Constants.INSTANCE;
+
+    @Inject
+    private TranslationService translationService;
+
+    @Inject
+    @DataField("list-group-item")
+    Div listGroupItem;
+
+    @Inject
+    @DataField("logIcon")
+    Span logIcon;
+
+    @Inject
+    @DataField("logTime")
+    Span logTime;
+
+    @Inject
+    @DataField("logTypeDesc")
+    Span logTypeDesc;
+
+    @Inject
+    @DataField("logInfo")
+    Span logInfo;
+
+    @Inject
+    @AutoBound
+    private DataBinder<TaskEventSummary> logSummary;
+
+    @PostConstruct
+    public void init() {
+        tooltip(logIcon);
+    }
+
+    @Override
+    public HTMLElement getElement() {
+        return listGroupItem;
+    }
+
+    @Override
+    public TaskEventSummary getValue() {
+        return this.logSummary.getModel();
+    }
+
+    @Override
+    public void setValue(final TaskEventSummary model) {
+        String iconClass = "list-view-pf-icon-sm kie-timeline-list-view-pf-icon-sm";
+        this.logSummary.setModel(model);
+        tooltip(logTime);
+        logTime.setTextContent(DateUtils.getPrettyTime(model.getLogTime()));
+        logTime.setAttribute("data-original-title",
+                             DateUtils.getDateTimeStr(model.getLogTime()));
+        //iconClass += " fa fa-user";
+        String logInfoContent = "(" + model.getUserId() + ") ";
```

Comment:
@cristianonicolai I added 'by process' in this situation, not sure what is the best solution here, regarding the task is 'added' because the process definition.

Reply:
I think the user id should be empty in that case, that would tell us that it is a system action. 
@mswiderski wdyt regarding https://github.com/kiegroup/jbpm/blob/master/jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java#L235 it doesn't look like this userId override to store the processId is quite right, any particular reason for doing that? Could we remove it?